quickwit-doc-mapper/src/lib.rs

```diff
@@ -137,7 +137,7 @@ pub fn default_doc_mapper_for_test() -> DefaultDocMapper {
 
 /// Returns a default `DefaultIndexConfig` for unit tests.
 #[cfg(any(test, feature = "testsuite"))]
-pub fn default_config_with_demux_for_tests() -> DefaultDocMapper {
```

Comment:
You should remove this function, I think it's useless now.

Reply:
The test `test_json_serialize()` uses this helper function.

Reply:
yes but it's useless...